public/index.html

```diff
@@ -6,6 +6,7 @@
     <link href="https://fonts.googleapis.com/css?family=Poppins|Revalia" rel="stylesheet">
     <link rel='stylesheet' type='text/css' href='/css/style.css'>
```

Comment:
If we remove the `/` from `/css/style.css` it will work with GitHub Pages as well 🎉